src/build/optimize-streams.ts

```diff
@@ -30,6 +30,11 @@ export type FileCB = (error?: any, file?: File) => void;
 export type CSSOptimizeOptions = {
   stripWhitespace?: boolean;
 };
+export interface OptimizeOptions {
+  htmlMinify?: HTMLMinifierOptions;
```

Comment:
Why not have the nested structure here that we do in BuildOptions? Then in #525 instead of adding a `jsCompile` property, you add `compile` to the `js` property. Or to go further, just get rid of this interface and pass BuildOptions to `getOptimizeStreams`.